homeassistant/components/vicare/__init__.py

```diff
@@ -54,7 +55,7 @@ class HeatingType(enum.Enum):
 def setup(hass, config):
     """Create the ViCare component."""
     conf = config[DOMAIN]
-    params = {"token_file": "/tmp/vicare_token.save"}
+    params = {"token_file": hass.config.path(STORAGE_DIR, "vicare_token.save")}
```

Comment:
Should we label this a breaking change?

Reply:
I don't know, help requested in the initial PR summary already... ping @oischinger  :)

Reply:
I don't know, help requested in the initial PR summary already... ping @oischinger  :)

Reply:
Since the token is autogenerated it should be completely transparent to the users.
Not breaking imho